src/main/java/de/wevsvirus/facades/ArztFacade.java

```diff
@@ -0,0 +1,45 @@
+package de.wevsvirus.facades;
+
+import com.github.javafaker.Faker;
+import de.wevsvirus.data.ArztRepository;
+import de.wevsvirus.model.ArztModel;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+
+import javax.annotation.Resource;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ArztFacade {
+
+    private Faker faker = new Faker();
+
+    @Resource
+    private ArztRepository arztRepository;
+
+    public List<ArztModel> findArztByPLZ(final String plz) {
+        String cleanPlz = StringUtils.trimAllWhitespace(plz);
+
+        List<ArztModel> aerzte = arztRepository.findByPlz(cleanPlz);
+        if (aerzte.size() > 0) {
+            return aerzte.stream()
+                    .limit(10L)
```

Comment:
Warum das Limit nicht gleich in die Quey einbauen wenn das wichtig ist?

Reply:
aus dem repo kommt kein normales listenobject. ja das limit hätte ich rausnehmen können aber kopieren müsst ich es trotzdem

Reply:
?? Du bekommst doch ne List aus dem Repository? Warum musste das kopieren?